csrankings-s.csv

```diff
@@ -1439,6 +1439,7 @@ Sokrates T. Pantelides,Vanderbilt University,https://my.vanderbilt.edu/sokratesp
 Sol M. Shatz,University of Illinois at Chicago,https://www.cs.uic.edu/~shatz,NOSCHOLARPAGE
 Solange O. Rezende,USP-ICMC,http://icmc.usp.br/pessoas?id=130627,e2GNtdIAAAAJ
 Solange Oliveira Rezende,USP-ICMC,http://icmc.usp.br/pessoas?id=130627,e2GNtdIAAAAJ
+Soledad Villar,Johns Hopkins University,https://engineering.jhu.edu/faculty/soledad-villar/,JBGlsDoAAAAJ

```

Comment:
<img width="1176" alt="Screen Shot 2024-08-05 at 10 06 23 AM" src="https://github.com/user-attachments/assets/0d026fb2-79dc-42f9-b970-eb60364496e6">


From this page: https://www.cs.jhu.edu/people/cs-affiliates/joint-and-secondary-faculty/